lib/ancestry/instance_methods.rb

```diff
@@ -116,6 +116,10 @@ def ancestry_changed?
       end
     end
 
+    def sane_ancestor_ids?
+      valid? || errors[self.ancestry_base_class.ancestry_column].blank?
+    end
```

Comment:
this will probably get merged back in when this concept changes. But probably good enough for now